gateway/justfile

```
@@ -14,15 +14,15 @@ install:
 
 # Run the tests
 test:
-    cargo test
+    cargo test --workspace
 
 # Format the source code
 format:
     cargo fmt
 
 # Verify code lints without applying any fix
 lint:
-    cargo clippy
+    cargo clippy --workspace --all-features
```

```suggestion
    cargo clippy --workspace --all-features --all-targets
```

And this should also be spread to `fix-lint` below (whether all-targets is in or not)